app/src/main/java/de/xikolo/controllers/course/CourseActivity.kt

```diff
@@ -168,6 +169,9 @@ class CourseActivity : ViewModelActivity<CourseViewModel>(), UnenrollDialog.List
         handleCourseDeepLinkTab(intent)
 
         updateViewPagerTab()
+
+        var recentCourses = RecentCoursesStorage()
```

Comment:
Again, that is the storage so please reflect it in the variable name. Also, this is probably a `val`.